src/arrayMethodReduce.js

```diff
@@ -5,7 +5,22 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let accumulator;
+    let i;
```

Comment:
We can add start values for accumulatorr and i variables and change them in if condition so we can avoid using else statement